### PR TITLE
doc(FEC 12554): create browsers support document + add missing player config documentation

### PR DESCRIPTION
**Pull request by giladna**
### Description of the Changes

Upgrade shaka player version and handle modified configuration api

Related PR: https://github.com/kaltura/playkit-js-dash/pull/202

Resolves [FEC 12554](https://kaltura.atlassian.net/browse/FEC 12554)

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: docs/playlist.md [hunk 1]
```diff
@@ -205,3 +227,7 @@ kalturaPlayer.loadPlaylist({playlistId: '56789'}, {options: {autoContinue: true}
   console.log(kalturaPlayer.playlist.options.autoContinue); // true
 });
 ```
+
+## Error Handling
+
+Error handling ise based on player errors which may result from BE requests or media issues
```
Comment (SivanA-Kaltura) [inline docs/playlist.md:233]:
```suggestion
Error handling is based on player errors which may result from Kaltura api requests or media issues 
```

File: docs/supportedPlatforms.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+
+## Kaltura Player Browsers Support 
+
+
+* Kaltura Player is being tested for playback & features on the stable and latest versions of the browsers below.
```
Comment (SivanA-Kaltura) [inline docs/supportedPlatforms.md:5]:
```suggestion
* Kaltura Player playback and features are tested on the stable and latest versions of the browsers below. 
```

File: docs/playlist.md [hunk 3]
```diff
@@ -173,8 +173,30 @@ kalturaPlayer.playlist.playItem(lastItemIndex);
...
+
+[Playlist Events](https://github.com/kaltura/kaltura-player-js/blob/master/src/common/playlist/playlist-event-type.js)
+
+
+ * `PLAYLIST_LOADED`
+ * `PLAYLIST_ITEM_CHANGED` (multiple)
+ * `PLAYLIST_ENDED`
+
+
+## Listening to Event before calling playlist API the first time
```
Comment (SivanA-Kaltura) [inline docs/playlist.md:188]:
```suggestion
## Listening to an Event before calling the playlist API the first time
```

File: docs/playlist.md [hunk 4]
```diff
@@ -173,8 +173,30 @@ kalturaPlayer.playlist.playItem(lastItemIndex);
 
 For the complete `playlist` API, see [PlaylistManager](./api.md#playlistmanager).
 
+## Playlist Events
+
+Once a playlist load is requested async calls will be called until the playlist object is ready for use. Inorder to use the api correctly you must be aware that the playlist is ready for use after the playtist is loaded (witing for `PLAYLIST_LOADED` event).
```
Comment (SivanA-Kaltura) [inline docs/playlist.md:178]:
```suggestion
Once a playlist load is requested, async api calls are made. You can use the playlist api after the playlist data is loaded (`PLAYLIST_LOADED` event).
```

File: docs/configuration.md [hunk 5]
```diff
@@ -100,6 +106,53 @@ The configuration uses the following structure:
...
+>
+> ##### Description: Defines the tracks custom labels configuration.
+
+##
+
+> ### config.dimensions
+>
+> ##### Type: [PKDimensionsConfig](https://github.com/kaltura/playkit-js/blob/master/docs/configuration.md#type-pkdimensionsconfig)
+>
+> ##### Description: Defines the player dimentions configuration.
```
Comment (SivanA-Kaltura) [inline docs/configuration.md:137]:
```suggestion
> ##### Description: Defines the player dimensions configuration.
```

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]

File: docs/playlist.md (hunk 4 above)
Comment (JonathanTGold) [reply docs/playlist.md:178]: The wording is unclear, what's the point?

File: docs/playlist.md (hunk 3 above)
Comment (JonathanTGold) [reply docs/playlist.md:188]: "for the first time" instead of "at the first time"

**Review by JonathanTGold — APPROVED** [review]

**Review by SivanA-Kaltura — APPROVED** [review]

**Pull request merged**